TheaterSchedule.DAL/Repositories/SettingsRepository.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Entities;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    class SettingsRepository : ISettingsRepository
+    {
+
+        private TheaterScheduleContext db;
+
+        public SettingsRepository(TheaterScheduleContext context)
+        {
+            this.db = context;
+        }
+
+        //Get
+        public Settings GetSettingsByPhoneId(string SettingsId)
+        {
```

Comment:
Maybe better will be create new account with default settings and return it?

Reply:
I've thought about it, but it's a get method, so we just get information not create.

Reply:
you right
just create new account and settings in store method